tests/python_yield/yield.py

```diff
@@ -0,0 +1,5 @@
+def example(first):
+    if False:
+        yield
+
+    yield first
```

Comment:
Can you add another test where the expression is long enough to break!

Reply:
Good idea! That caught some issues with my implementation :)